HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRPlayer.cs

```diff
@@ -640,6 +652,11 @@ private void GetPlayerWirebugs()
             return;
         }
 
+        bool IsBlocked = Process.Memory.Deref<int>(
```

Comment:
```suggestion
        bool isBlocked = Process.Memory.Deref<int>(
```

Variables should start with lowercase, here's the [C# Coding Conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#naming-conventions)